src/pydap/model.py

```diff
@@ -287,18 +288,15 @@ class StructureType(DapType, Mapping):
     """A dict-like object holding other variables."""
 
     def __init__(self, name, attributes=None, **kwargs):
-        DapType.__init__(self, name, attributes, **kwargs)
+        super(StructureType, self).__init__(name, attributes, **kwargs)
 
         # emulate a simple ordered dict
         self._keys = []
         self._dict = {}
 
     def __repr__(self):
         return '<%s with children %s>' % (
-            self.__class__.__name__, ', '.join(map(repr, self.keys())))
-
-    def __contains__(self, child):
```

Comment:
It's still a good idea to implement `__contains__`. The `collections.Mapping` implementation defaults to iterating over every element.